# Cvičenie 2

In [1]:
#kniznica na kreslenie grafov
import matplotlib.pyplot as plt
#numericko matematicka kniznica
import numpy as np

## Príklad 2.1
>Ukážte, že vlnová dĺžka pre neutróny je rovná
>
>$\frac{\lambda}{2 \pi}\approx 4,5/\sqrt{E}~,~~~~~\textrm{E v MeV, $\lambda$ v fm}$
>
>Zhruba pre aké energie je vlnová dĺžka väčšia ako rozmer jadra?
>
>Ak je $\lambda$ omnoho väčšie ako rozmer jadra, aký majú rozptýlené neutróny v rozptylovom experimente na jadre moment hybnosti? 

#### riešenie
$$ p=\frac{h}{\lambda} \Rightarrow \qquad \frac{\lambda}{2\pi}=\frac{\hbar}{\sqrt{2 m E}} $$ 

približný faktor 4.5 dostaneme po dosadení


In [2]:
200/np.sqrt(2000)

4.47213595499958

## Príklad 2.2
>Aká je elektrostatická energia rovnomerne nabitej gule? 

#### riešenie
$U=\frac{1}{4\pi\epsilon_0}\int\frac{Q(r)}{r}dQ=\frac{3}{5}\frac{1}{4\pi\epsilon_0}\frac{Z^2 e^2}{R}$


## Príklad 2.3

>a) Zo známych hmotností atómov $^{15}$O m$_{\textrm{O}}=	15.003065$u a $^{15}$N m$_{\textrm{N}}=15.0001088982$u
> vypočítajte rozdiel medzi väzbovými energiami príslušných jadier. (Hmotnosti jadier, ako
> aj väzbové energie možno nájsť prehľadne na [stránke]( http://nucleardata.nuclear.lu.se/database/masses/).
>
>b) Pri uvážení, že rozdiel väzbových energií $^{15}$O a $^{15}$N je daný len Coulombovou energiou, 
vypočítajte polomery daných jadier.

#### riešenie

**a)** všeobecný vzťah pre atómovu hmotnosť
$$m c^2 (A,Z)= Z m_H + N m_N -B/c^2, $$

kde B je väzbová energia.

Rozdiel väzbových energií kyslíka a dusíka je:
$$ \Delta B= B_{^{15}{\textrm{O}}}-B_{^{15}{\textrm{N}}}=m_{^{15}{\textrm{N}}}-m_{^{15}{\textrm{O}}}+m_H-m_N$$
Vypočítajme

In [3]:
# hmotnosti v u
mO=15.003065
mN=15.0001088982
mH=1.007825
mn=1.00866491588
# u v jednotkach MeV/c^2
u=931.49410242 
# rozdiel vo vazbovych energiach
deltaB=mN-mO+mH -mn  
# v jednotkach MeV
deltaB*u 

-3.535968081601055

**b)**
$$\Delta B=-\frac{3}{5}\frac{(2Z-1)}{4\pi\epsilon_0 R},$$
kde Z je náboj kyslíka.

Vyjadrime si R, dostaneme

In [4]:
# konstanty
alpha=1/137.035999084
htrans_c=197.326980 # priblizne 200 Mev fm

# R vo fermi
3/5*(alpha*htrans_c*(2*8-1))/3.5219956700646633

3.6796413495278886

a pre $r_0=R/A^{1/3}$

In [5]:
# r_0
3.6796413495278886/15**(1/3)

1.4920214639395284

čo je trocha viac ako štandardne uvádzaná hodnota pre $r_0$ 1,1 až 1,3 fm.

## Príklad 2.4
>V jednoduchom **Fermiho modeli jadra** aproximujeme jadro ako sústavu voľných nukleónov viazaných  
v trojrozmernej pravouhlej potenciálovej jame s objemom $V=R^3$, kde $R$ je polomer jadra. Vo vnútri jamy je potenciál nulový 
a na povrchu rovný hodnote $V_0$. Pre zjednodušenie výpočtu sa v tomto modeli
používajú hodnoty pre jednotlivé energie nukleónov $E_n=\frac{\pi^2\hbar^2}{2mL^2}( n_x^2+  n_y^2+n_z^2 )$, t.j. energie 
z takzvanej nekonečnej potenciálovej jamy 
$V_0\longrightarrow\infty$.
>
>a) Aký počet nukleónov sa nachádza v energetickom intervale $(E, dE)$.
>
>b) Akú najvyššiu energiu má nukleón v základnom stave (t.j. aká je Fermiho energia sústavy)?
>
>c) Vyčíslite Fermiho energiu sústavy. Ak uvážime, že separačná energia nukleónov je zhruba 8 MeV, 
čo môžeme povedať o hĺbke potenciálovej jamy, t.j. akú typickú hodnotu má $V_0$?
>
>d) Nájdite celkovú kinetickú energiu nukleónov.
>
>e) Rozvinte výraz pre celkovú kinetickú energiu do druhého rádu v parametri $(Z-N)/A$.
>
>f) Aká je väzbová energia na jeden nukleón, pre jadrá so $Z=N$?

#### riešenie

**a)** Vnútri gule s polomerom $n=\sqrt{n_x^2+n_y^2+n_z^2}$ je $N_E=2\times \frac{4}{3}\pi n^3$ stavov. Berieme do úvahy len 
kladné $n$, teda 1/8 gule.
Vyjadrime $n$ pomocou energie a zderivujme podla $E$. Dostávame tak počet stavov v energetickom intervale $(E,E+dE)$:


$$ dN_E=\frac{\sqrt{2 m^3}L^3}{\pi^2\hbar^3\pi^2}\sqrt{E}dE$$


**b)** Ak máme vypočítané N$_E$ z predošlej podúlohy, potom Fermiho energiu jednoducho nájdeme ako $E_f=\frac{1}{2m}\left( 3\pi^2\hbar^{3}\rho \right)^{2/3}$, kde $\rho=N_E/L^3$ je hustota protónov alebo neutrónov v jadre.


**c)** $\rho\approx 0.14/2$, pretože berieme zvlášť protóny a zvlášť
 neutróny. Potom máme $E_f$ v $MeV$: 

In [6]:
1/(2000)*(3*np.pi**2*htrans_c**3*0.14/2)**(2/3)

31.648682553773728

Typická hĺbka jamy je zhruba $V_0\approx 31.6+8 \approx 40$ MeV.

**d)** Využije sa, že protóny a neutróny majú separátne potenciálové jamy. A  použijeme $\rho=\frac{Z}{\frac{4}{3}\pi r_0^3 A}$. Potom $E=\int^{E_f}_0E dN_E=\frac{3}{10 m}\frac{\hbar^2}{r_0^2}\left(\frac{9\pi}{4} \right)^{2/3}A \frac{N^{5/3}+Z^{5/3}}{A^{5/3}}$. 

**e)** Rozpíšeme $N=A/2(1-(Z-N)/(2A))$ a $Z=A/2(1+(Z-N)/(2A))$, použijeme T. rozvoj a dostaneme $E\approx\frac{3}{10 m}\frac{\hbar^2}{r_0^2}\left( \frac{9\pi}{8}  \right)^{2/3}\left( A +\frac{5}{9}\frac{(N-Z)^2}{A}\right)$ 

**f)**
$\frac{3}{10 m}\frac{\hbar^2}{r_0^2}\left( \frac{9\pi}{8}  \right)^{2/3}$ sa dá vyjadriť v MeV ako 

In [7]:
3/(10*938)*(htrans_c)**2/1.2**2*(9*np.pi/8)**(2/3)

20.066137915770746

potom väzbová energia na jeden nukleón je $40-20\approx 20$MeV